app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt

```diff
@@ -63,14 +67,20 @@ fun Week(
             state = statePager
         ) { page ->
             DayCard(
-                clickEnabled = date == dates[page],
                 date = dates[page],
                 day = days[page],
                 onClick = {
-                    if (tasksFiltered.isEmpty()) {
-                        stateDialog = true
-                    } else {
-                        stateShowCards = !stateShowCards
+                    if (date != dates[page]) {
+                        scope.launch {
+                            statePager.scrollToPage(page)
+                        }
+                    }
+                    else {
```

Comment:
This is not our used style for conditional statements. `else` should be after closing `if` body bracket, in the same line. Ctrl+Alt+L